### ISSUE TITLE
File name optimization, add "defer", console greeting to confirm connectivity

### ISSUE THREAD

**Issue by klutchdev** [h5bp/html5-boilerplate#2323]
**I'm submitting a ...**

- [ ] bug report
- [x] feature request
- [ ] other (Please do not submit support requests here (below))

* * * 

### **Hey Everyone!!!**
I **LOVE** this tool, it's amazing for devs such as myself that write mostly in **Vanilla Js**, especially when frameworks are overkill for the task at hand.
My feature request(s) are simple- I'd be happy to help implement them if any of it is deemed worthy!

## **1. Change the "main.js" file name to "app.js"**
> _Vs Code and many other IDEs/code editors default to "app.js" as the "default" Javascript file_

## **2. Add "defer" to the source Js file script tag:**
```html
<script defer src="app.js"></script>
``` 
> _This allows all DOM elements to load before executing any Js scripts_

## **3. Add 1 line of code to main.js (_or app.js, if its changed_):**

```javascript
console.log(' √ Js file connected! Thanks for using HTML5 Boilerplate 🔥 ');
```
> _This is just a neat way of confirming that our HTML and Js files are good to go!_

## **4. Change the "main.css" file name to "style.css"**
```html
<link rel="stylesheet" href="style.css">
```
> _Vs Code and many other IDEs/code editors default to "style.css" as the "default" css file import_

**Comment by klutch-kid** [comment]
I can be reached via **klutchdevelopment@gmail.com** in addition to github.
Thanks!!!

**Comment by klutchdev** [comment]
Should I close this?

**Comment by roblarsen** [comment]
I'll follow up when I have time. So just let it sit for now. I'm not sold on any of these charges, but I also want to properly address the suggestions.

**Comment by klutchdev** [comment]
Oh ok! No problem , I just don't want to cause anyone a hard time. No rush on this end at all, I just didn't want to have the issues tab glaring with something I opened and not be proactive about it. Cheers! 🔥

**Comment by roblarsen** [comment]
No worries at all. Sometimes this stuff moves quickly, other times not so much.

**Comment by klutchdev** [comment]
I don't plan on doing anything for a very long time except advance my skills and continue to tutor a couple students!!! I had no idea you have a couple books out, I'll be scooping those up ASAP. Also, It's great to see a fellow dev from Mass... I'm by Springfield, but I have played a lot of shows out that way. Anything you need, just gimme a shout no rush for any reason, nothing is broken haha

**Comment by roblarsen** [comment]
@vltansky it might be cool to add the file names as an option to the create script.

Spoiler: I'm leaning towards app.js, but am hesitant about changing the name of main.css, just because of the project that hosts it. I guess that's a discussion point. Does it matter if the upstream project has a different name? I think styles.css is a better name, for sure. Maybe I'll tweet about it. There'd be no problems with having main.css as a dependency (as a project) and then building it with whatever name we want. We could even add that as an option to the main.css project. hrm. I'll try to move the ball on this over the weekend. 

🤔

**Comment by klutchdev** [comment]
Sounds good. This is my first feature request, I appreciate it

**Comment by coliff** [comment]
I vote in favour of using the **style.css** file name. A user's final compiled CSS will include our 'main.css', but it would also include  their own custom styles and maybe styles from other plugins etc - so the final CSS in their project isn't just 'main.css'.

**Comment by klutchdev** [comment]
aye 👍🏻

**Comment by mohamedmehdigara** [comment]
Hi @klutchdev. I have forked this repository and added changes like you requested. Now what should I do?

**Comment by klutchdev** [comment]
oh!! uhhh, I think it's up to @roblarsen ? He was going to think about some changes @coliff, @vltansky, and myself talked about so I'd say just hang tight

**Comment by roblarsen** [comment]
closed via #2341 and #2342 Thanks for the suggestion @klutchdev !

**Comment by klutchdev** [comment]
welcome
